alluxio-python-library/alluxio/alluxio_file_system.py

```diff
@@ -156,6 +159,52 @@ def listdir(self, path):
                 f"Error when listing path {path}: error {e}"
             ) from e
 
+    def get_file_status(self, path):
+        """
+        Gets the file status of the path.
+
+        Args:
+            path (str): The full ufs path to get the file status of
+
+        Returns:
+            list of dict: A list containing dictionaries, where each dictionary has:
```

Comment:
1. if it's get file status instead of list status, can we return only one dictionary telling all the file info, instead of list of dict which suits list status API call
2. Not quite sure if Python prefers a dict or whether python has struct / object class? @ChunxuTang any idea? what are the better ways to represent a file info?  @ssyssy may be we can learn from fsspec and see how they represent a file info

Reply:
The same changes can be apply to list_status call

Reply:
@LuQQiu  checked fsspec the result is one dictionary.